libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.spec.ts

```diff
@@ -56,6 +57,43 @@ describe('ModalWrapperComponent', () => {
     });
   });
 
+  describe('sizing', () => {
+    beforeEach(() => {
+      spectator = modalWrapperTestBuilder
+        .flavor('modal')
+        .withEmbeddedInputComponent()
+        .build();
+    });
+    afterEach(() => {
+      spectator.fixture.destroy();
+    });
+
+    it('should trigger `onScrollElementResize` when embeddded component resizes', (done) => {
+      setTimeout(() => {
+        // @ts-ignore
+        expect(spectator.component.resizeObserverService.observe).toHaveBeenCalledWith(
+          // @ts-ignore
+          spectator.component.getEmbeddedComponentElement(),
+          jasmine.any(Function)
+        );
+        done();
+      });
+    });
+
+    it('should clean up resize observer of embedded component, on destroy', (done) => {
+      spectator.component.ngOnDestroy();
+
+      setTimeout(() => {
+        // @ts-ignore
+        expect(spectator.component.resizeObserverService.unobserve).toHaveBeenCalledWith(
+          // @ts-ignore
+          spectator.component.getEmbeddedComponentElement()
```

Comment:
If you need to access private members you can do the following, which still gets type checked:
```suggestion
        expect(spectator.component['resizeObserverService'].unobserve).toHaveBeenCalledWith(
          spectator.component['getEmbeddedComponentElement()']
```

Reply:
Shouldn't this be spectator.component['getEmbeddedComponentElement']() then or does spectator.component['getEmbeddedComponentElement()'] actually works?